services/provider/server/server.go

```diff
@@ -647,7 +647,7 @@ func (s *OCSProviderServer) GetStorageClaimConfig(ctx context.Context, req *pb.S
 	}
 	var extR []*pb.ExternalResource
 
-	storageRequestHash := getStorageRequestHash(req.StorageConsumerUUID, req.StorageClaimName)
+	storageRequestHash := util.CalculateMD5Hash(req.StorageClaimName)
```

Comment:
@rewantsoni We cannot apply this change, it breaks backport compatibility with existing deployments.